packages/editable-html/src/serialization.jsx

```diff
@@ -356,7 +356,13 @@ serializer.deserialize = function deserialize(html) {
   return null;
 };
 
-export const htmlToValue = html => serializer.deserialize(html);
+export const htmlToValue = html => {
+  try {
+    return serializer.deserialize(html);
+  } catch {
+    return {};
```

Comment:
@CarlaCostea can you please also log the error?

Reply:
sure